src/framework/theme/components/form-field/prefix.directive.ts

```diff
@@ -0,0 +1,7 @@
+import { Directive } from '@angular/core';
+
+@Directive({
+  selector: '[nbPrefix]',
```

Comment:
I think we may some other `nbPrefix` in the future
```suggestion
  selector: '[nbFormControlPrefix]',
```